service/controller/resource/pvc/local_pvc.go

```diff
@@ -0,0 +1,57 @@
+package pvc
+
+import (
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+)
+
+func newLocalPVCs(customObject v1alpha1.KVMConfig, pvsList *corev1.PersistentVolumeList) ([]*corev1.PersistentVolumeClaim, error) {
+	var persistentVolumeClaims []*corev1.PersistentVolumeClaim
+
+	for i, workerKVM := range customObject.Spec.KVM.Workers {
+		for _, hostVolume := range workerKVM.HostVolumes {
+			for _, pv := range pvsList.Items {
+				if pv.ObjectMeta.Labels["mount-tag"] == hostVolume.MountTag {
+					persistentVolumeClaim := &corev1.PersistentVolumeClaim{
+						TypeMeta: metav1.TypeMeta{
+							Kind:       "PersistentVolumeClaim",
+							APIVersion: "v1",
+						},
+						ObjectMeta: metav1.ObjectMeta{
+							Name: key.LocalWorkerPVCName(key.ClusterID(customObject), key.VMNumber(i)),
+							Labels: map[string]string{
+								"app":      key.WorkerID,
+								"cluster":  key.ClusterID(customObject),
+								"customer": key.ClusterCustomer(customObject),
+								"node":     "",
```

Comment:
Not sure how here passing the Worker ID

Reply:
I made an attempt there to pass the worker ID please take a look at it @tfussell